Automattic/wp-calypso
# Implement the trail map A/B/n test together with the overridable feature flags.

Resolve #89689

## Proposed Changes

This PR implements the A/B/n test for the 4 variations of the trail map experiment, given the latest direction according to p1714093659520579-slack-C06P2UC516Y.

Abacus profile: 21737-explat-experiment

1. Control
2. Copy only
3. Structure only
4. Copy and structure

## Testing Instructions

* The plans grid should work intact without any feature flag given.
* Verify that the structure-only & copy-and-structure variants will indeed alter the features grid layout by enabling the relevant feature flags or assigning yourself to the corresponding variations on Abacus.
* The copy-wise variants are harder to verify now, so it's good enough to just verify the logic by reviewing.

## Pre-merge Checklist
